features/post-meta.feature

```diff
@@ -139,3 +139,95 @@ Feature: Manage post custom fields
       """
       My\New\Meta
       """
+
+  @pluck
+  Scenario: Multi-dimensional values can be plucked.
+    Given a WP install
+    And an input.json file:
+      """
+      {
+        "foo": "bar"
+      }
+      """
+    And I run `wp post meta set 1 meta-key --format=json < input.json`
+
+    When I run `wp post meta pluck 1 meta-key foo`
+    Then STDOUT should be:
+      """
+      bar
+      """
+
+  @pluck @pluck-deep
+  Scenario: Multi-dimensional values can be plucked at any depth.
+    Given a WP install
+    And an input.json file:
+      """
+      {
+        "foo": {
+          "bar": {
+            "baz": "some value"
+          }
+        },
+        "foo.com": {
+          "visitors": 999
+        }
+      }
+      """
+    And I run `wp post meta set 1 meta-key --format=json < input.json`
+
+    When I run `wp post meta pluck 1 meta-key foo.bar.baz`
```

Comment:
I think the preferred approach should be to accept an arbitrary number of positional arguments with each own representing one level.

The benefit is that you can have both periods and spaces be part of the keys:
```
# foo -> bar -> baz
wp post meta pluck 1 meta-key foo bar baz

# foo.bar -> baz
wp post meta pluck 1 meta-key foo.bar baz

# foo bar -> baz
wp post meta pluck 1 meta-key "foo bar" baz
```

Any mechanism which needs to to provide all levels in one positional argument will force you to have one arbitrary character be impossible to use as a key identifier.